packaging/windows/readme.md

```diff
@@ -7,17 +7,11 @@
 -- Use Windows' default console window (especially on Windows 10)  
 - Install [Go](https://golang.org/dl/) 1.5.3 (soon to switch to 1.6)  
 - Open a command console and make a directory for cloning the repo, e.g.:  
-`cd c:\`  
-`mkdir work`  
-`cd work`  
-`mkdir src`  
-`cd src`  
-`mkdir github.com`  
-`cd github.com`  
-`mkdir keybase`  
-`cd keybase`  
+`cd ~`  
```

Comment:
No, I'm using cmd, as indicated by "Use Windows' default console window". I know everyone hates anything windows specific, so you can get rid of that part and make it bash-only if you want, but I'm just saying "~" contradicts "Use Windows' default console window" and we may as well keep both.